tests/e2e/utils/CheReporter.ts

```diff
@@ -24,6 +24,7 @@ import { OAUTH_CONSTANTS } from '../constants/OAUTH_CONSTANTS';
 import { REPORTER_CONSTANTS } from '../constants/REPORTER_CONSTANTS';
 import { PLUGIN_TEST_CONSTANTS } from '../constants/PLUGIN_TEST_CONSTANTS';
 import { inject, injectable } from 'inversify';
+const chromeHar = require('chrome-har')
```

Comment:
Could it be performed using 'import' ?

Reply:
Using `import` causes an error:
![image](https://github.com/eclipse/che/assets/83611742/a1c4f6e0-8848-493f-bc0e-6e1c487205c1)

There is no `@types` package for `chrome-har`, so I currently use the package like so:
https://github.com/eclipse/che/blob/88e2b95970fb07fb975dbd304c4124cf744380fa/tests/e2e/utils/CheReporter.ts#L29-L30

WDYT?

Reply:
I see, in this case I am ok with both solutions. We could create own "types" file but I don`t think it is necessary for just one dependency

Reply:
Sounds good, I have left it at:
```
 // @ts-ignore: No module declaration file 
 import * as chromeHar from 'chrome-har'; 
```